hugo/package.json

```diff
@@ -6,38 +6,34 @@
   "private": true,
   "scripts": {
     "clean:static": "shx rm -rf ./static/showcase ./static/playground ./static/libs",
-    "copy:libs:prepare": "shx mkdir -p ./static/libs/monaco-editor-workers/workers",
-    "copy:monaco-editor-wrapper": "npm run copy:libs:prepare && shx cp -fr ../node_modules/monaco-editor-wrapper/bundle ./static/libs/monaco-editor-wrapper",
+    "copy:libs:prepare": "shx mkdir -p ./static/libs/monaco-editor-workers/workers && shx mkdir -p ./static/libs/langium-website-foundation",
```

Comment:
We don't need to create a directory for the langium-website-foundation, nothing is copied there.

Reply:
Thanks for spotting that.